src/modules/dashboard/views/ViewDashboard.vue

```diff
@@ -107,6 +107,7 @@
       <PendingDataTable :transaction="pendingTransactions" />
       <div class="text-txs text-gray-400 mt-10 mb-2"><b class="text-gray-700 uppercase">{{$t('dashboard.recentTransactions')}}</b> </div>
       <MixedTxnDataTable :selectedGroupType="transactionGroupType.CONFIRMED" :transactions="recentTransactions" @openDecryptMsg="openDecryptMsgModal"></MixedTxnDataTable>
+      <a :href="linkToExplorer()" target=_new v-if="searchedTransactions.length==10"><div class="text-right text-xs text-blue-primary mt-3" >View more...</div></a>
```

Comment:
target is wrong, should be "_blank"

Reply:
Hi Chun Lam, changed the target to "_blank".